### PR TITLE
Initial frontend commit

### PR DESCRIPTION
**Pull request by balinskia**
Created mocked views using vue-cli template. Created:

- Request new loan
- Loans list
- My investments view
- Single investment view

Created reusable layout components: TopPanel and InvestmentPanel

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# getline-frontend
+
+> This is the frontend app for the Getline Distributed P2P Lending Platform
+
+## Build Setup
+
+``` bash
+# install dependencies
+npm install
```
Comment (q3k) [inline frontend/README.md]: Mind if we use/recommend yarn instead of npm?

File: frontend/server/README [hunk 2]
```diff
@@ -0,0 +1,3 @@
+In this folder we have files representing the sample output of our backend.
+Those files are used to mock backend.
+The name of files represent the url's suffix.
```
Comment (q3k) [inline frontend/server/README]: s/url's suffix/backend method/

Comment (q3k) [reply frontend/server/README]: Generally these LGTM as mocks, but the actual data format will almost certainly change when the metabackend comes to life :).

File: frontend/src/components/Invest/LoanList/LoanList.vue [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          </md-table-row>
+        </md-table-body>
+      </md-table>
+    </div>
+  </div>
+</template>
+
+<script>
+import NoLoans from './NoLoans';
+import UserScore from '../../common/UserScore';
```
Comment (q3k) [inline frontend/src/components/Invest/LoanList/LoanList.vue]: Is there a way to do imports in JS that doesn't involve a ton of '../../../../'? Some sort of absolute imports?

**Comment by q3k** [comment]
Please address comments.

Excellent job, thank you.

File: frontend/server/getGatheringLoans.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  time: '1 Month',
+  procentFunded: '47%',
+  procentNeeded: '53%',
+}, {
+  id: 3,
+  userName: 'Kyle Spencer',
+  userScore: 'B',
+  amountNeeded: '123.45 USD',
+  time: '1 Month',
+  procentFunded: '47%',
```
Comment (q3k) [inline frontend/server/getGatheringLoans.js]: s/procent/percent/

File: frontend/README.md (hunk 1 above)
Comment (balinskia) [reply frontend/README.md]: sure, no problem, changed to yarn

File: frontend/src/components/Invest/LoanList/LoanList.vue (hunk 3 above)
Comment (balinskia) [reply frontend/src/components/Invest/LoanList/LoanList.vue]: Yes:
```
import UserScore from '@/components/common/UserScore'
```
I will aviod: "../.../../../../"

**Pull request merged**